### ISSUE TITLE
Support `test_` prefixes

### ISSUE THREAD

**Issue by brettcannon** [jasonrudolph/significant-other-vscode#1]
Fairly common in Python projects.

**Comment by jasonrudolph** [comment]
Thanks for the suggestion, @brettcannon. :bow:

I'm not very familiar with the Python ecosystem, so could you link to a Python project that's considered to be idiomatic in the way it organizes its tests? For example, would https://github.com/psf/requests be a reasonable choice?

**Comment by md2perpe** [comment]
Yes, requests is a reasonable choice. It's a widely used library and has a standard structure.

**Comment by jasonrudolph** [comment]
Thanks, @md2perpe. I'm not sure how soon I'll be able to look at this, but I'm interested in finding a way to support this.

**Comment by md2perpe** [comment]
I made a solution that works quite well: https://github.com/md2perpe/significant-other-vscode/commit/a786d04b627f055b5f1f1dbe048e0b22501f7117

It takes the patterns from the configuration, one pattern for the ordinary source file and one pattern for the test file, and matches the current relative file path with them and translates from one to the other.

I think that the idea is good, but the pattern syntax and the matching might need some work.

**Comment by brettcannon** [comment]
@jasonrudolph the requests example is good enough.
